walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java

```diff
@@ -41,7 +41,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<Task> tasks = new ArrayList<>();
+   List<Task> tasks = new ArrayList<>();
```

Comment:
oh no! the spacing! Can you revert this file since I think this was a mistake?
Or run `make format`.

Reply:
Fixed